### DISCUSSION TITLE
Introduce a new Blank Page style filter

### DISCUSSION THREAD

**Discussion by Noracsa2000** [proginosko/LeechBlockNG#457] [Ideas]
I would like to have a new filter for ‘Apply filter instead of blocking page’ that simply displays a white screen like the ‘Blank Page’. This should create the strong blocking effect of the Blank Page, but with the functional difference that the site is automatically usable again after the blocking time has expired.

My use case: I ‘block’ certain phases of the day moderately with the greyscale filter. Within these phases, there are short ‘forced breaks’ with a harder filter, which is intended to block usage completely. I currently use blur 32px for this, which works functionally. However, it is neither optimal nor aesthetically pleasing that greyscale becomes colour again at that moment and I can still see all sites blurred. It would be better if the sites were simply blank white and then, afterwards, greyscale again.

There are certainly many other scenarios in which a complete block like with the ‘Blank Page’ is temporarily desired, without the loaded site being lost by loading about:blank, but being available again normally after the block time.

**Comment by proginosko** [comment]
It would be easy to add "fade (100%)" to the list of filter options, which would be equivalent to a blank page.

**Reply by Noracsa2000** [reply]
I would really appreciate a "fade (100%)" filter option. As I understand the fade filters, they generally fade to the background colour of the page. So this option would be equivalent to a blank page only for sites with a white background, but fading to a different colour whenever the background colour is not white. This is why I'd rather have a dedicated blank page filter. However, if implementing "fade (100%)" is much easier and thus more realistic to implement at all, I'm all for it :)

**Reply by proginosko** [reply]
I think "fade (100%)" will be better for most users, since many users use 'dark' themes for websites. I doubt they would appreciate a completely white page all of a sudden. 🙂

I've added the option in version 1.6.5.

**Reply by Noracsa2000** [reply]
Thank you so much! :) I've tested it for a day and it works like a charm. As I also use an auto dark mode switcher, I can confirm that this implementation with fade (100%) is indeed better than a blank white page.
